src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
   // extract the json from the response
   const result = response.data.choices[0].text ?? ''
+
```

Comment:
### Line 26
## CodeGuard Suggestions
**Suggestion:** Remove the lines that console log the input json as these are not needed
**Reason:** Result should not be logged for security reasons